dffml/operation/datetime.py

```diff
@@ -0,0 +1,39 @@
+import datetime
+from ..df.base import op
+from ..df.types import Definition
+
+current_datetime = Definition(name="current_datetime", primitive="generic")
+date_time = Definition(name="date_time", primitive="generic")
+
+
+@op(name="dffml.datetime", outputs=current_datetime)
+def date_time():
+    """
+    A function to generate current date and time.
+
+    Parameters
+    ----------
+    output:
+        Current date and time.
+    """
+    current_datetime = datetime.datetime.now()
+    return current_datetime
```

Comment:
```suggestion
    return {
        "date_time_obj": current_datetime
    }
```

Reply:
Related discusion on if we fix https://github.com/intel/dffml/issues/1350 then we don't need this suggestion (can be done after merging this PR, could change back after 1350 is addressed)

Reply:
Example doctest as unittest is dffml/source/dataset/iris.py, consider using `type()`